prime-router/src/testIntegration/kotlin/datatests/TranslationTests.kt

```diff
@@ -375,6 +390,30 @@ class TranslationTests {
                 } else null
             }
 
+            if (!config.conditionFiler.isNullOrBlank()) {
+                if (config.conditionFiler.equals("pruneUnwanted")) {
```

Comment:
We should probably document this special setting somewhere, maybe in `prime-router/src/testIntegration/resources/datatests/README.md`.

Reply:
> We should probably document this special setting somewhere, maybe in `prime-router/src/testIntegration/resources/datatests/README.md`.

good heads up, I overlooked the readme, will fix it